### DISCUSSION TITLE
Migrate to named exports

### DISCUSSION THREAD

**Discussion by dai-shi** [pmndrs/zustand#1506] [Ideas] [answered]
#1475 is raised and we released v4.2.0.

@barelyhuman said:
> I think if we slowly let people used named imports instead that would be a good solution in the long run.

It's unfortunate that the entire ecosystem will still be taking time to support ES default export fully. Recently, I gave up going this road and made migration to named exports in one of my projects: [proxy-memoize](https://github.com/dai-shi/proxy-memoize)

Let's deprecate default export in v4 and migrate to named exports.
Hopefully, we have enough time before v5 which will remove the deprecated one.

How do people think?

**Comment by judicaelandria** [comment]
100%!! I think that would be awesome

**Comment by dai-shi** [comment, marked as answer]
Opened #1514.
